packages/gatsby/src/query/types.ts

```diff
@@ -25,3 +27,9 @@ export interface IGraphQLRunnerStatResults {
   uniqueFilterPaths: number
   uniqueSorts: number
 }
+
+export type PageContext = any
```

Comment:
this should be `unknown` because we can never know statically what will be in pageContext

Reply:
Oh, nice. TIL.